### ISSUE TITLE
Paranoid Verifiable Documentation

### ISSUE THREAD

**Issue by swilson223** [phatworx/devise_security_extension#180]
Hey, 

Im new to this particular gem. ive used devise in the past, this new app requires some beefy security and i think this is the best to handle what im looking for.. is there any better documentation on how to implement this into my app? the documentation provided sort of lacs substance in getting it off the ground.

**Comment by dmz006** [comment]
Do you have device running yet? If so it's just an extension of that with configuration details in the initializer & models using device using the configuration on https://github.com/phatworx/devise_security_extension.  If you don't have device working yet check out http://railscasts.com/episodes/209-devise-revised and http://railscasts.com/episodes/210-customizing-devise

**Comment by swilson223** [comment]
Yes Devise is running, working every thing on that front is happy - happy.. im just looking for more substance on implimenting the paranoid verifiable module and want to use it as the examples, where an admin can provide a code upon password reset or lock an account and need the code to reactivate it ie after a suspension..
